### DISCUSSION TITLE
(API) How to make loading function?

### DISCUSSION THREAD

**Discussion by piw-piw** [ScratchAddons/ScratchAddons#4655] [Help] [answered]
#### JS API for addons

How to make loading function, when user changes to costume editor?

I want make this:
https://github.com/ScratchAddons/ScratchAddons/issues/4507#issue-1206223204

**Comment by TheColaber** [comment, marked as answer]
You can use the addon.tab.waitForElement function (its in the scratchaddons.com docs) to wait for an element that's in the costume editor.
